kotlinx-coroutines-core/common/src/CoroutineContext.common.kt

```diff
@@ -20,7 +20,6 @@ public expect fun CoroutineScope.newCoroutineContext(context: CoroutineContext):
 @InternalCoroutinesApi
 public expect fun CoroutineContext.newCoroutineContext(addedContext: CoroutineContext): CoroutineContext
 
-@PublishedApi
```

Comment:
Please leave it on actuals as well.
It is used not for the sake of inlining but in order to access it with the help of `@Suppress` from a different KLib. `@PublishedApi` guarantees a stable unmangled name here